android/quest/src/main/java/org/smartregister/fhircore/quest/ui/profile/ProfileScreen.kt

```diff
@@ -131,9 +135,17 @@ fun ProfileScreen(
       if (!fabActions.isNullOrEmpty()) {
         ExtendedFloatingActionButton(
           contentColor = Color.White,
-          text = { Text(text = fabActions.first().display.uppercase()) },
+          text = { fabActions.first().display?.let { Text(text = it.uppercase()) } },
```

Comment:
This safety null check using the `let` scope function is not required as the `display` property is not nullable